### ISSUE TITLE
A unique URL does not display when searching references on classic

### ISSUE THREAD

**Issue by aazaff** [paleobiodb/web_app#57]
If I search for a reference number (e.g., 59892), the URL of the displayed page just says "https://www.paleobiodb.org/classic" rather than a specific URL. We should be returning unique URLs for each of these.

![screen shot 2017-05-26 at 11 22 08 am](https://cloud.githubusercontent.com/assets/10422595/26503310/cfa86358-4205-11e7-93ae-b6e6bfdb48c1.png)

**Comment by aazaff** [comment]
Why is this not a bug?

**Comment by jpjenk** [comment]
With limited labels to choose from in the GitHub issue tracker, we try to reserve "bug" for critical failure other really serious stuff. Unless it is marked "enhancement", everything is a bug of some description.

That being said, this issue has been merged with #61 and is now considered a high priority. We will figure out a way to show the payload in the URL for taxon and other resources.

This went away not because of a flaw but as a result of the new framework. Remember the web_app is not PBDB classic, that has gone away completely. It currently borrows some of the same code and templates - and we made a poor choice in naming a core module "classic" in development - but it runs very differently. Over time more parts will be replaced and there will be no legacy code. Technically speaking, closest thing to what used to be called PBDB Classic is actually Fossilworks.
